python/svix/receiver.py

```diff
@@ -29,7 +27,7 @@ def verify(self, data: t.Union[bytes, str], headers: t.Dict[str, str]) -> t.Dict
         msg_id = headers.get("svix-id")
         msg_signature = headers.get("svix-signature")
         msg_timestamp = headers.get("svix-timestamp")
-        if not (msg_id and msg_timestamp and msg_timestamp):
+        if not (msg_id and msg_timestamp and msg_signature):
```

Comment:
👀

Reply:
?

Reply:
I see you sneaking a bugfix in :P

Reply:
Ah yeah, because it was actually discovered by the type checks so I let it slide. :)